modules/fs.js

```diff
@@ -26,27 +22,26 @@
  *   console.error('test.txt already exists.');
  * }
  */
-
+var log = require("ringo/logging").getLogger(module.id);
 var arrays = require('ringo/utils/arrays');
+var {PosixPermissions} = require('ringo/utils/files');
 include('io');
 include('binary');
 
-var File = java.io.File,
-    FileInputStream = java.io.FileInputStream,
-    FileOutputStream = java.io.FileOutputStream;
+var Files = java.nio.file.Files,
+    FileSystems = java.nio.file.FileSystems;
+
+const FS = FileSystems.getDefault();
+const getPath = function(path) {
+    return FS.getPath(path);
+};
```

Comment:
@grob: I tried some stuff and const works like expected. The value does not change and there is no exception. If a module is reloaded, the const is re-evaluated, but that is what I would expect from the runtime: https://github.com/botic/const-refs-test and https://www.youtube.com/watch?v=mUlR8Nr6ogw